docs-kits/kits/Business Partner Kit/page_changelog.md

```diff
@@ -11,6 +11,22 @@ sidebar_position: 1
 
 All notable changes to this Kit will be documented in this file.
 
+## [2.0.0] - 2023-08-25
+
+<h3>Added</h3>
```

Comment:
Instead of using html tags, please use the markdown headline syntax `###`. Docusaurus will translate this into the proper html tag and styling